wandb/sdk/wandb_run.py

```diff
@@ -2134,6 +2134,27 @@ def _swap_artifact_name(self, artifact_name: str, use_as: Optional[str]) -> str:
     def _detach(self) -> None:
         pass
 
+    def link_artifact(
+        self,
+        artifact: Union[public.Artifact, Artifact],
+        portfolio_path: str,
+        aliases: List[str],
+    ) -> None:
+        portfolio, project, entity = wandb.util._parse_entity_project_item(
+            portfolio_path
+        )
+
+        # if local artifact, then use client_id
+        # if public artifact, use the real server_id as artifactID
+        if self._backend and self._backend.interface:
+            if not self._settings._offline:
+                self._backend.interface.publish_link_artifact(
+                    self, artifact, portfolio, aliases, entity, project,
+                )
+            else:
+                # TODO: implement offline mode + sync
```

Comment:
Are you planning on getting this part in this PR?

Reply:
No, not in this PR.